packages/mdc-toolbar/README.md

```diff
@@ -136,6 +287,85 @@ The provided modifiers are:
 
 | Class                                | Description                             |
 | -------------------------------------| --------------------------------------- |
-| `mdc-toolbar--fixed`                 | Make toolbar fixed to top of screen.    |
+| `mdc-toolbar--fixed`                 | Makes toolbar fixed on top and have persistent elavation |
+| `mdc-toolbar--flexible`              | Makes toolbar first row has flexible space |
+| `mdc-toolbar--fixed-lastrow-only`    | Makes last row of toolbar anchored on top |
 | `mdc-toolbar__section--align-start`  | Makes section aligns to the start.      |
 | `mdc-toolbar__section--align-end`    | Makes section aligns to the end.        |
+
+
+## JS Usage
+
+### Including in code
+
+#### ES2015
+
+```javascript
+import {MDCToolbar, MDCToolbarFoundation} from 'mdc-toolbar';
+```
+
+#### CommonJS
+
+```javascript
+const mdcToolbar = require('mdc-toolbar');
+const MDCToolbar = mdcToolbar.MDCToolbar;
+const MDCToolbarFoundation = mdcToolbar.MDCToolbarFoundation;
+```
+
+#### AMD
+
+```javascript
+require(['path/to/mdc-toolbar'], mdcToolbar => {
+  const MDCToolbar = mdcToolbar.MDCToolbar;
+  const MDCToolbarFoundation = mdcToolbar.MDCToolbarFoundation;
+});
+```
+
+#### Global
+
+```javascript
+const MDCToolbar = mdc.toolbar.MDCToolbar;
+const MDCToolbarFoundation = mdc.toolbar.MDCToolbarFoundation;
+```
+
+### Instantiation
+
+```javascript
+import {MDCToolbar} from 'mdc-toolbar';
+
+const toolbar = new MDCToolbar(document.querySelector('.mdc-toolbar'));
+```
+
+### Method
+
+| Method Signature | Description |
+| --- | --- |
+| `updateAdjustElementStyles() => void` | sets `AdjustElement` proper `margin-top`. |
+
+### Event
```

Comment:
I think the table looking is just another way for quick referencing, which can be useful for a quick lookup. I suggest we keep it.